continuous_integration/environment-3.10.yaml

```diff
@@ -38,7 +38,7 @@ dependencies:
   - sortedcollections
   - tblib
   - toolz
-  - tornado<6.2
+  - tornado=6.2
```

Comment:
```suggestion
  - tornado>=6.2
```

Reply:
This came from #6991, should we update these pins for _all_ environments? (not just 3.10)

Reply:
yes probably all the environments need it, otherwise it's fighting the pin from distributed==2022.10.2

https://github.com/dask/distributed/pull/6991#pullrequestreview-1095039021

Reply:
Done, hopefully I caught all the spots.